lib/pig_latin_cases.rb

```diff
@@ -1,5 +1,3 @@
-require 'exercise_cases'
```

Comment:
And it simply doesn't break anything... if that dependency goes away, no problem, it is a documentation value at this point.